account_cutoff_base/models/account_cutoff.py

```diff
@@ -0,0 +1,355 @@
+# © 2013-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+
+class AccountCutoff(models.Model):
+    _name = 'account.cutoff'
+    _rec_name = 'cutoff_date'
+    _order = 'cutoff_date desc'
+    _inherit = ['mail.thread']
+    _description = 'Account Cut-off'
+
+    @api.multi
+    @api.depends('line_ids', 'line_ids.cutoff_amount')
+    def _compute_total_cutoff(self):
+        for cutoff in self:
+            tamount = 0.0
+            for line in cutoff.line_ids:
+                tamount += line.cutoff_amount
+            cutoff.total_cutoff_amount = tamount
+
+    @api.model
+    def _default_move_label(self):
+        cutoff_type = self.env.context.get('cutoff_type')
+        label = ''
+        if cutoff_type == 'accrued_expense':
```

Comment:
Why not using a list or dictionnary like:
```
CUTOFF_TYPES = [('accured_expense', 'Accrued Expense')]
next(item[1] for item in toto if item[0] == cutoff_type
```
It can be seen as less readable, but it is easier to extend right?
And we shouldn't forget to return it as translatable.

Reply:
:open_mouth: 
keep it simple and use a dict which perfectly match this case:
```
@property
def cutoff_type_label_map(self):
    return {
        'accrued_expense': _('Accrued Expense'),
        'accrued_revenue': _('Accrued Revenue'),
        'prepaid_revenue': _('Prepaid Revenue'),
        'prepaid_expense': _('Prepaid Expense'),
    }

@api.model
def _default_move_label(self):
    cutoff_type = self.env.context.get('cutoff_type')
    return self.cutoff_type_label_map.get(cutoff_type, '')

# for the field
def _selection_cutoff_type(self):
    return list(self.cutoff_type_label_map.items())
```

Reply:
@simahawk In the beginning this function was used to set a default in https://github.com/OCA/account-closing/pull/61/files#diff-06ebd0f0e6a30443bc5c088d5efddd24R62